test/corpus/expressions.txt

```diff
@@ -4149,3 +4149,53 @@ asm ("nop")
     text: (string))
   (asm
     text: (string)))
+
+=============
+uninitialized
+=============
+
+uninitialized Int32
+a = uninitialized Int32
+@b =
+uninitialized
+Int32*
+
+---
+
+(expressions
+  (call
+    method: (identifier)
+    arguments: (argument_list
+      (constant)))
+  (assign
+    lhs: (identifier)
+    rhs: (uninitialized_var
+      (constant)))
+  (assign
+    lhs: (instance_var)
+    rhs: (uninitialized_var
+      (pointer_type
+        (constant)))))
+
+===========
+out keyword
+:language(crystal)
+===========
+
+@[MyAnnotationIs(out @of_this_world)]
+method.call(out _)
```

Comment:
Would be good to add tests for the other argument list types, especially together with blocks